docs/pages/admin-guides/infrastructure-as-code/managing-resources/user-and-role.mdx

```diff
@@ -367,8 +367,6 @@ created and granted the correct roles.
 If you have UI access, connect to your Teleport cluster Web UI, open the
 management panel, and select the "Users" tab.
```

Comment:
There's no management panel anymore. This should guide towards Access -> Users instead.

Reply:
Changed